specs/spec-1-kudos.md

```diff
@@ -1,106 +1,310 @@
+Here are some user stories or use cases for the Kudos widget based on the provided text:
 
-- create or remove categories
-- adjust min expire time
+1. As a **visitor** (not logged in), I want to view the Kudos list, so I can see the various accomplishments and recognitions in the community.
+2. As a **visitor**, I want to view Kudos details, so I can learn more about a specific Kudo and understand its context.
+3. As a **visitor**, I want to share Kudos (list and deep link to a Kudo detail), so I can promote achievements and encourage others to engage with the community.
+4. As a **logged-in user**, I want to comment on Kudos, so I can express my thoughts and engage with the community.
+5. As a **logged-in user**, I want to react to Kudos, so I can show my appreciation and support for the accomplishments.
+6. As a **verified human**, I want to create new Kudos, so I can recognize and celebrate the contributions and achievements of others in the community.
+7. As a **verified human**, I want to receive new Kudos, so I can be acknowledged for my contributions and achievements.
+8. As an **administrator**, I want to delete Kudos, so I can remove inappropriate or unwanted content from the platform.
+9. As an **administrator**, I want to change settings, such as creating or removing categories, adjusting expiration time of Kudos, and deleting Kudos, so I can maintain a positive and productive environment for the community.
 
-### NEAR Social Use Cases
+## Additional Requirements
 
-We define the following features which will use the Social DB smart contract with UI developed in Near Social:
+The following features will use the Social DB smart contract with the Kudos UI developed in Near Social.
 
+### Create a kudo
+
+- Tag any IAH verified account
+- Notify the account of the Kudo via the Notifications API
+- Send account SBT Badge for each Kudo.
+- Display the Kudo as a post in the activity feed.
+- Check that creator and reciever are registered with IamHuman.
 - Account receiving a Kudo will get receive Near Social notification via Notifications API.
-- Upvote a Kudo.
-  Any human can upvote a Kudo (only when Kudo is not expired). One account can upvote any Kudo only once.
-  Upvotes will be stored in the Social DB or in the Kudo SBT issuer (TODO: check what's better).
+
+### Interact with existing Kudo
+
+- Upvote a Kudo
+  - Any human can upvote a Kudo (only when Kudo is not expired). 
+  - One account can upvote any Kudo only once.
+  - Upvotes will be stored in the Social DB or in the Kudo SBT issuer.
 - Comment on a Kudo
-  Any human can comment on a Kudo (only even if a Kudo expired). Comments will be implemented using existing Near Social widget.
-- Near Social widget to show a Kudo with all comments.
-- Follow Button to receive notification on new comments for a given Kudo.
+  - Any human can comment on a Kudo (only even if a Kudo expired). 
+  - Comments will be implemented using existing Near Social widget.
```

Comment:
shall we be able to respond on comments? (so having a threaded comments)

Reply:
Sure, threaded comments are looking good on near.org.